encass-annotations/src/main/gateway/config/stored-passwords.properties

```diff
@@ -0,0 +1 @@
+jdbc_con_pass=7layer
```

Comment:
use the name as **internal-users-password**

Reply:
Done